e2eshark/requirements.txt

```diff
@@ -12,9 +12,8 @@ accelerate
 auto-gptq
 optimum
 azure-storage-blob
-# install nightly build of torch_mlir 
--f https://github.com/llvm/torch-mlir-release/releases/expanded_assets/dev-wheels
-torch-mlir
+# install nightly build of torch_mlir, if on Linux (no macOS or Windows nightly builds)
+torch-mlir ; sys_platform == "linux" -f https://github.com/llvm/torch-mlir-release/releases/expanded_assets/dev-wheels
```

Comment:
I did not know you could do this. Cool!

Reply:
Yep!

* https://peps.python.org/pep-0496/ 
* https://stackoverflow.com/questions/29222269/is-there-a-way-to-have-a-conditional-requirements-txt-file-for-my-python-applica